spacy/pipeline/edit_tree_lemmatizer.py

```diff
@@ -116,6 +123,7 @@ def __init__(
 
         self.cfg: Dict[str, Any] = {"labels": []}
         self.scorer = scorer
+        self.store_activations = store_activations  # type: ignore
```

Comment:
`store_activations` is a property of `TrainablePipe`. The setter guarantees that we always get `List[str]` when we use the getter. So, `for activation in self.store_activations` is safe, because `self.store_activations` will never return a `bool`.

The issue is that we MyPy is currently not able to handle setters with a different type of argument than the property getter and therefore the `type: ignore` is needed here. It's a long thread, but upstream now wants to support this use case, but not one has implemented it yet.

https://github.com/python/mypy/issues/3004

Reply:
Aaah, right, I hadn't seen the setter bit, thanks!